net/oic/src/api/oc_ri.c

```diff
@@ -587,6 +586,8 @@ oc_ri_invoke_coap_entity_handler(struct coap_packet_rx *request,
     response_buffer.response_length = 0;
     response_buffer.code = oc_status_code(OC_STATUS_FORBIDDEN);
     success = false;
+  } else if (response_buffer.code >= BAD_REQUEST_4_00) {
```

Comment:
I think it would be more idiomatic by writing `response_buffer.code >= oc_status_code(OC_STATUS_BAD_REQUEST)`

Reply:
Yeah, I did consider it. However, given that the error code mappings are present and done within this very file, I decided that it was worth it.